Source/WebCore/platform/graphics/ContentsFormat.cpp

```diff
@@ -39,13 +39,13 @@ std::optional<DestinationColorSpace> contentsFormatExtendedColorSpace(ContentsFo
     switch (contentsFormat) {
     case ContentsFormat::RGBA8:
         return std::nullopt;
-#if HAVE(IOSURFACE_RGB10)
+#if ENABLE(PIXEL_FORMAT_RGB10) && ENABLE(DESTINATION_COLOR_SPACE_EXTENDED_SRGB)
     case ContentsFormat::RGBA10:
```

Comment:
But `ContentsFormat` defines `RGBA10` when `ENABLE(PIXEL_FORMAT_RGBA16F)` is true. This switch statement has to handle `RGBA10` anyway when `ENABLE(PIXEL_FORMAT_RGBA16F)` is true.

Reply:
Well, it would be a compile error if there ever were a combination of ENABLE(PIXEL_FORMAT_RGB10) but not ENABLE(DESTINATION_COLOR_SPACE_EXTENDED_SRGB), so if that ever comes up, someone will have to decide what behavior they want and implement it. For now, there is no reason to do that.